ddtrace/contrib/gevent/__init__.py

```diff
@@ -5,11 +5,8 @@
 The integration patches the gevent internals to add context management logic.
 
 .. note::
-    If :ref:`ddtrace-run<ddtracerun>` is being used set ``DD_GEVENT_PATCH_ALL=true`` and
-    ``gevent.monkey.patch_all()`` will be called as early as possible in the application
-    to avoid patching conflicts.
-    If ``ddtrace-run`` is not being used then be sure to call ``gevent.monkey.patch_all``
-    before importing ``ddtrace`` and calling ``ddtrace.patch`` or ``ddtrace.patch_all``.
+    If ``ddtrace-run`` is not being used then be sure to import ``ddtrace``
```

Comment:
I know docs are versioned, but should we think about keeping a section for `DD_GEVENT_PATCH_ALL` around for a bit since not everyone will be on latest?